authorization.go

```diff
@@ -32,7 +32,7 @@ type Authorization struct {
 // AuthorizationParams is a set of params for creating entity.
 type AuthorizationParams struct {
 	PaymentMethodToken     string                 `json:"payment_method_token"`
-	CreditCardCvv          string                 `json:"credit_card_cvv"`
+	CreditCardCvv          string                 `json:"credit_card_cvv,omitempty"`
```

Comment:
Minor: CVV is abbreviature, so in golang this field should look like `CreditCardCVV`